com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Settings/RenderPipelineSettings.cs

```diff
@@ -94,6 +94,8 @@ internal static RenderPipelineSettings NewDefault()
                 supportedLitShaderMode = SupportedLitShaderMode.DeferredOnly,
                 supportDecals = true,
                 supportDecalLayers = false,
+                supportSurfaceGradient = false,
```

Comment:
should be true by default. We want it for new project but keep it false for old project (can be done wit the migration step)